core-services/prow/02_config/generate-boskos.py

```diff
@@ -175,6 +175,9 @@
     'gcp-qe-quota-slice': {
         'us-central1': 30,
     },
+    'gcp-autorelease-qe-quota-slice': {
+        'us-central1': 4,
```

Comment:
Do we have other region which have enough quota to run ci testing ?

Reply:
It looks no for now. Shall I set up one?

Reply:
okay, if not yet, let us use the the existing one.